fbs/client/response.fbs

```diff
@@ -0,0 +1,25 @@
+// SPDX-FileCopyrightText: 2023 Siemens AG
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// Author: Michael Adler <michael.adler@siemens.com>
+
+include "header.fbs";
+
+namespace generated.plugin.client;
+
+enum ResponseStatus: byte {
+  // The request was denied and may not proceed.
+  Deny = 0,
```

Comment:
Maybe we need another "error" dimension here: permanent (`Deny`) vs. transient (`Retry`)?